rustler/src/types/atom.rs

```diff
@@ -197,6 +194,44 @@ unsafe impl Send for Atom {}
 /// The only overhead is checking that the atoms have been created (an atomic integer read).
 ///
 #[macro_export]
+macro_rules! atoms {
+    {
+        $(
+            $( #[$attr:meta] )*
+            $name:ident $( = $str:expr )*,
```

Comment:
In newer Rust versions, you can use `?` instead of `*`, which also disallows `bla = "a" = "b"`.

Reply:
Also, I think the comma should be optional, in line with other usages, so `atoms!{ ok }` should work. To do this, it should be enough to change the rule to
```rust
        $(
            $( #[$attr:meta] )*
            $name:ident $( = $str:expr )?
         ),* $(,)?
```

(see https://stackoverflow.com/questions/43143327/how-to-allow-optional-trailing-commas-in-macros/43143459#43143459)

Reply:
Neat.

Reply:
Done.